docs/guide/concepts.md

```diff
@@ -16,48 +16,48 @@ sidebarDepth: 2
 
 `type State = any`
 
-State 表示 Model 的状态数据，通常表现为一个 javascript 对象（当然它可以是任何值）；操作的时候每次都要当作不可变数据（immutable data）来对待，保证每次都是全新对象，没有引用关系，这样才能保证 State 的独立性，便于测试和追踪变化。
+State represents the state data of the Model, usually represented as a javascript object (of course it can be any type). It is treated as immutable data, ensuring that each time it is a new object. No Quote relationships so that State independence is guaranteed, making it easy to test and track changes.
```

Comment:
`State` represents the state of the `Model`, that is usually represented as a javascript object (of course it can be any type). It should be treated immutable, ensuring that each time it is a new object without any reference relationships so that `State` independent. That will make easy to test and track changes.